.env.example

```
@@ -26,3 +26,15 @@ AIRFLOW_SIMPLE_ADMIN_PASSWORD=admin
 AIRFLOW_SIMPLE_VIEWER_PASSWORD=viewer
 PIPELINE_BACKEND_BASE_URL=http://backend:8080
 PIPELINE_ARTIFACT_ROOT=/opt/airflow/artifacts
+
+# dev 배포용 환경변수
+# 애플리케이션 런타임 환경변수는 Render Dashboard 또는 Blueprint(render.yaml)에서 설정
+# GitHub Secrets는 GitHub Actions 배포 워크플로우에서 Render API 호출 등에만 사용되며, Render 컨테이너에 직접 주입되지 않음
+SPRING_PROFILES_ACTIVE=dev
+SPRING_DATASOURCE_URL=jdbc:postgresql://<neon-host>:5432/<dbname>?sslmode=require
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SPRING_DATASOURCE_URL key should go before the SPRING_PROFILES_ACTIVE key

(UnorderedKey)